app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/menu/DrawerMenu.kt

```diff
@@ -24,31 +27,13 @@ class DrawerMenu @Inject constructor(
     private lateinit var drawerLayout: DrawerLayout
     private lateinit var currentNavigationItem: DrawerNavigationItem
 
-    fun setup(
-            toolbar: Toolbar,
-            drawerLayout: DrawerLayout,
-            navigationView: NavigationView,
-            actionBarDrawerSync: Boolean = false
-    ) {
-        this.drawerLayout = drawerLayout
-        if (actionBarDrawerSync) {
```

Comment:
Is it okay to remove this flag?

Reply:
No problem because this method is no longer mandatory.